service/ztests/query-runtime-error-log.yaml

```
@@ -0,0 +1,20 @@
+script: |
+  LAKE_EXTRA_FLAGS='-log.path=service.log' source service.sh
+  zed create -use -q test
+  echo '{ts:1,union:"hi"((int64,string))}' | zed load -q -
```

Nit: You can use a simpler input here (though it'll give you a different error).
```suggestion
  echo {} | zed load -q -
```

---

Fine by me as long as we get the error.